packages/cli/test/functional/test_site/testPageNavWithOnlyTitle.md

```diff
@@ -0,0 +1,11 @@
+<frontmatter>
+  layout: default.md
+  pageNav: 1
```

Comment:
Since pageNav is 1 and there's a H1 heading, is this example containing only the title?

Reply:
That's a good point. It contains title and the H1 heading. Sorry about that! Let me fix it now

Reply:
thanks yongliang!

It should look like this:

![image](https://user-images.githubusercontent.com/46486515/208372250-d6b011dc-89b9-4119-b2e0-4cfe9f2a4305.png)